core/src/main/resources/error/error-classes.json

```diff
@@ -76,6 +76,13 @@
       "Could not load Protobuf class with name <protobufClassName>. <explanation>."
     ]
   },
+  "CANNOT_MODIFY_CONFIG" : {
+    "message" : [
+      "Cannot modify the value of a Spark config: <key>.",
+      "See also 'https://spark.apache.org/docs/latest/sql-migration-guide.html#ddl-statements'."
```

Comment:
This one clashes with https://github.com/apache/spark/pull/39820, correct?

Reply:
Oh... yes.
Will keep this in mind fix this in #39820 after merging.

Reply:
It has been merged, @itholic please, adjust this PR.

Reply:
Got it! Thanks for the notify

Reply:
Adjusted, thanks!